### ISSUE TITLE
Cannot upgrade to v2.0.0

### ISSUE THREAD

**Issue by jjsquady** [blockloop/scan#34]
Hi, I`m trying to update to version 2.0.0, but go get -u points always do 1.3.0 

What I`m doing wrong?

Thanks :D

**Comment by emarj** [comment]
You are doing nothing wrong, the `go.mod` file has not been correctly updated for this. I will now submit a pull request to fix this.

**Comment by emarj** [comment]
As a temporary workaround you can use
```
github.com/blockloop/scan@445acc200003cd77fb417a453b6ece6f14fa3c79
```
For example

```bash
go get -u github.com/blockloop/scan@445acc200003cd77fb417a453b6ece6f14fa3c79
```

Once the PR is merged the import would became
```go
import "github.com/blockloop/scan/v2"
```

**Comment by blockloop** [comment]
Sorry we completely missed this. I merged #46 and re-tagged so everything should be good!
I also fixed notifications so that we won't miss future issues.
